src/python/WMCore/JobStateMachine/ChangeState.py

```diff
@@ -416,6 +416,8 @@ def recordInCouch(self, jobs, newstate, oldstate, updatesummary = False):
                                   "site": job.get("location", None),
                                   "cms_location": job["fwjr"].getSiteName(),
                                   "exitcode": job["fwjr"].getExitCode(),
+                                  "eos_log_url": job["fwjr"].getLogURL(),
```

Comment:
Isn't getLogURL a LogArchive method? Here you're looking for it in a Report object, no? Same for the file above.

Reply:
Sorry while I was merging with master branch, made a mistake. Yes these are Report methods.